### PR TITLE
Fix events view

### PR DESCRIPTION
**Pull request by vivianliang**

### CODE REVIEW

File: core/views/events.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  def get(self, request, event_id=None, *args, **kwargs):
+    if event_id is not None:
+      try:
+        event = Event.objects.get(id=event_id)
+      except Event.DoesNotExist:
+        raise Exception('invalid user id')
+      data = EventSerializer(event).data
+
+    else:
```
Comment (mgeist) [inline core/views/events.py]: what do you think about just making these 2 separate views, eventsview get (no id, returns all (or paginated eventually)) vs eventview get which takes an id? feels a little more separation of concerns

File: core/views/events.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+class EventsView(APIView):
+
+  def get(self, request, event_id=None, *args, **kwargs):
+    if event_id is not None:
+      try:
+        event = Event.objects.get(id=event_id)
+      except Event.DoesNotExist:
+        raise Exception('invalid user id')
```
Comment (mgeist) [inline core/views/events.py]: user -> event

Comment (vivianliang) [inline core/views/events.py]: heh heh

File: appfiles/app/scripts/controllers/main.coffee [hunk 3]
```diff
@@ -9,18 +9,18 @@
...
   @awesomeThings = [
     'HTML5 Boilerplate'
     'AngularJS'
     'Karma'
   ]
-  $scope.getEvent = ->
-    promise = Event.get 1
+
+  $scope.getEvents = ->
+    promise = Events.get()
```
Comment (mgeist) [inline appfiles/app/scripts/controllers/main.coffee]: curious - do you prefer this style
```
promise = Events.get()
promise.then (events) ->
  etc
```
or
```
Events.get().then (events) ->
  etc
```
either is fine with me, just want to know if we want to pick one to run with

Comment (vivianliang) [inline appfiles/app/scripts/controllers/main.coffee]: i don't care :(

**Pull request merged**